DataHub/Operation/03_ManageHTMLTemplates.md

```diff
@@ -0,0 +1,296 @@
+
+
+
+## Include Smarty templates
+
+You can include Smarty templates into your HTML template.    
+For detailed information about Smarty templates, see [Smarty](https://www.smarty.net/).    
+Always use the delimiters *{}* as separator for the Smarty templates.  
+
+#### Prerequisites
+
+The *HTML Templates for ETL* plugin has been installed.
+
+#### Procedure
+
+> [Info] If you want to include Smarty templates into a new HTML template, follow the steps described in the [Create an HTML template](#create-an-html-template) procedure before proceeding to the steps below.
+
+*DataHub > Settings > Tab HTML TEMPLATES > Select HTML template*   
+
+![Edit HTML template](../../Assets/Screenshots/DataHub/Settings/HTMLTemplates/EditHTMLTemplate.png "[Edit HTML template]")
+
+1. Click the textarea and enter the desired Smarty template(s).
+
+    > [Info] For detailed information about the Smarty template structure and elements, see [Smarty](https://www.smarty.net/).
+
+2. If desired, [include HTML text](#include-html-text) or display a [preview of the HTML template](#preview-an-html-template).
+
+3. Click the [SAVE] button in the upper right corner.   
+    The HTML template has been saved. The *HTML template* view is closed.  
+
+
+
+### Include attribute variables
+
+You can include variables for attributes via Smarty templates into your HTML template.   
+Always use the *$* sign to introduce an attribute variable.
+
+#### Prerequisites
+
+The *HTML Templates for ETL* plugin has been installed.
+
+#### Procedure
+
+*DataHub > Settings > Tab HTML TEMPLATES > Select HTML template*   
+
+![Edit HTML template](../../Assets/Screenshots/DataHub/Settings/HTMLTemplates/EditHTMLTemplate.png "[Edit HTML template]")
+
+> [Info] If you want to include variables into a new HTML template, follow the steps described in the [Create an HTML template](#create-an-html-template) procedure before proceeding to the steps below.
+
+1. Click the textarea and, if desired, edit or add the Smarty template(s) and/or HTML text for the template.
+
+2. Include a variable in the textarea by entering the corresponding variable inside the delimiters *{}* and starting with the *$* sign. You can include one of the following variables:
+    - **entity**   
+        Use the entity variable to include a variable referring to an attribute of a certain entity within the system, for instance *{$entity.id}*.
+    - **tenant**   
+        Use the tenant variable to include a variable referring to an attribute of the platform tenant, for instance *{$tenant.shortName}*.  
+
+    > [Info] The data fields of an entity you can refer to are described in the API documentation. To include a deeper level of the data field, enter a point *.* at a time, for instance *{$entity._pim_tax_zone.id}*.    
+
+3. If desired, include the destination language and/or the destination channel to the attribute variable value by adding the following suffixes at the end of the variable name:
```

Comment:
irgendwo könnte man erwähnen dass das die gleichen felder wie in der api-beschreibung sind bzw. genau den gleichen regeln folgt

Reply:
so wie in der info?
>> The data fields of an entity you can refer to are described in the API documentation. To include a deeper level of the data field, enter a point *.* at a time, for instance *{$entity._pim_tax_zone.id}*.